File: Java/Program/DPR.java
/*

Saya Nabilla Assyfa Ramadhani [2205297] 
mengerjakan Latihan
dalam mata Desain dan Pemograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan 
seperti yang telah dispesifikasikan. 
Aamiin

*/


class DPR{
    // variabel yang hanya dapat diakses dalam kelas itu saja
    private String id;
    private String nama;
    private String bidang;
    private String partai;
    char jenis_kelamin;

    // variabel yang dapat diakses oleh main
    public DPR(){

    }

    public DPR (String id, String nama, String bidang, String partai, char jk){
        // konstruction dengan inisialisasi
        this.id = id;
        this.nama = nama;
        this.bidang = bidang;
        this.partai = partai;
        this.jenis_kelamin = jk;
    }

    // Fungsi dan prosedur untuk get dan set ID
    public void setId (String id){
        this.id = id;
    }

    public String getId (){
        return this.id;
    }

    // Fungsi dan prosedur untuk get dan set Nama
    public void setnama (String nama){
        this.nama = nama;
    }

    public String getnama (){
        return this.nama;
    }

    // Fungsi dan prosedur untuk get dan set Nama Bidang
    public void setbidang (String bidang){
        this.bidang = bidang;
    }

    public String getbidang (){
        return this.bidang;
    }

    // Fungsi dan prosedur untuk get dan set Nama Partai
    public void setpartai (String partai){
        this.partai = partai;
    }

    public String getpartai (){
        return this.partai;
    }

    // Fungsi dan prosedur untuk get dan set Jenis Kelamin
    public void setJK (char jk){
        this.jenis_kelamin = jk;
    }

    public char getJK (){
        return this.jenis_kelamin;
    }

};

File: PHP/Program/DPR.php
<!-- 

Saya Nabilla Assyfa Ramadhani [2205297] 
mengerjakan Latihan 1
dalam mata Desain dan Pemograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan 
seperti yang telah dispesifikasikan. 
Aamiin

 -->

<?php

class DPR{
    private $id = '';
    private $nama ='';
    private $bidang = '';
    private $partai = '';
    private $jenis_kelamin = '';
    private $foto_profil = ''; 

    public function __construct ($id ='', $nama='', $bidang='', $partai='', $jk='', $foto_profil=''){
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->jenis_kelamin = $jk;
        $this->foto_profil = $foto_profil; 
    }

    // Metode set untuk atribut foto profil
    public function setFotoProfil($foto_profil){
        $this->foto_profil = $foto_profil;
    }

    // Metode get untuk atribut foto profil
    public function getFotoProfil(){
        return $this->foto_profil;
    }

    // Metode set dan get untuk id
    public function setId($id){
        $this->id = $id;
    }

    public function getId(){
        return $this->id;
    }

    // Metode set dan get untuk nama
    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    // Metode set dan get untuk bidang
    public function setBidang($bidang){
        $this->bidang = $bidang;
    }

    public function getBidang(){
        return $this->bidang;
    }

    // Metode set dan get untuk partai
    public function setPartai($partai){
        $this->partai = $partai;
    }

    public function getPartai(){
        return $this->partai;
    }

    // Metode set dan get untuk jenis kelamin
    public function setJenisKelamin($jk){
        $this->jenis_kelamin = $jk;
    }

    public function getJenisKelamin(){
        return $this->jenis_kelamin;
    }  

}

?>
